### PR TITLE
15.0 mig account financial report

### PR DESCRIPTION
**Pull request by koenloodts**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please include #847, but the CIs are red right now here.

**Comment by yankinmax** [comment]
@koenloodts are you working on it? Can I help to speed up a migration?

File: account_financial_report/tests/test_journal_ledger.py [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+                "date_to": self.fy_date_end,
+                "company_id": self.company.id,
+                "journal_ids": [(6, 0, self.journal_sale.ids)],
+                "move_target": "all",
+            }
+        )
+        data = wiz._prepare_report_journal_ledger()
+        res_data = self.JournalLedgerReport._get_report_values(wiz, data)
+        self.check_report_journal_debit_credit(res_data, 250, 250)
+        self.check_report_journal_debit_credit_taxes(res_data, 0, 300, 0, 50)
```
Comment (yankinmax) [inline account_financial_report/tests/test_journal_ledger.py]: @pedrobaeza can you help pls to solve issues?

**Comment by yankinmax** [comment]
@koenloodts I want to help, but can't open PR on your branch. Maybe you can give me access and we'll contribute together?
@pedrobaeza there is a change in taxes calculation in v15 probably. I can't make tests working. Can you pls help or point to a change if you know.

File: account_financial_report/tests/test_journal_ledger.py (hunk 1 above)
Comment (pedrobaeza) [reply account_financial_report/tests/test_journal_ledger.py]: That's part of the migration duties, to adapt things to make them still to work. I can't say without doing it myself, sorry.

**Comment by HaraldPanten** [comment]
Hi @koenloodts will you continue the PR?

Regards.

**Comment by HaraldPanten** [comment]
I close this PR as it has been superseded by https://github.com/OCA/account-financial-reporting/pull/868

**Pull request closed without merging**